.build/flags.mk

```diff
@@ -3,7 +3,9 @@ PKGS ?= $(shell glide novendor)
 LIST_PKGS ?= $(shell go list ./... | grep -v /vendor/)
 
 # Many Go tools take file globs or directories as arguments instead of packages.
-PKG_FILES ?= *.go auth config dig internal metrics modules service testutils tracing ulog
+ROOT_PKG_FILES := $(wildcard *.go)
+PKG_NAMES := auth config dig internal metrics modules service testutils tracing ulog
```

Comment:
this should be something like `go list | grep -v vendor`, rather than enumerating them all by hand